{"implements":["java.lang.Thread.UncaughtExceptionHandler"],"javadoc":"A class which handles exceptions that occur off of the main test thread.  Exceptions can be\n reported to this class, which will later be checked by {@link AbstractGenericTest}.","static":false,"extends":"java.lang.Object","methods":[{"javadoc":"","static":false,"name":"<init>","comment":"","params":[],"throws":[],"return":{"type_long":"void","comment":"","type_short":"void"}},{"javadoc":"","static":true,"name":"registerHandler","comment":"","params":[],"throws":[],"return":{"type_long":"void","comment":"","type_short":"void"}},{"javadoc":"","static":true,"name":"handle","comment":"","params":[{"type_long":"java.lang.Thread","name":"thread","comment":"","type_short":"Thread"},{"type_long":"java.lang.Throwable","name":"t","comment":"","type_short":"Throwable"}],"throws":[],"return":{"type_long":"void","comment":"","type_short":"void"}},{"javadoc":"","static":true,"name":"clear","comment":"","params":[],"throws":[],"return":{"type_long":"void","comment":"","type_short":"void"}},{"javadoc":"","static":true,"name":"enable","comment":"","params":[],"throws":[],"return":{"type_long":"void","comment":"","type_short":"void"}},{"javadoc":"","static":true,"name":"disable","comment":"","params":[],"throws":[],"return":{"type_long":"void","comment":"","type_short":"void"}},{"javadoc":"","static":true,"name":"isEnabled","comment":"","params":[],"throws":[],"return":{"type_long":"boolean","comment":"","type_short":"boolean"}},{"javadoc":"","static":true,"name":"getExceptions","comment":"","params":[],"throws":[],"return":{"type_long":"java.util.List<generic.test.TestExceptionTracker>","comment":"","type_short":"List"}},{"javadoc":"","static":true,"name":"hasException","comment":"","params":[],"throws":[],"return":{"type_long":"boolean","comment":"","type_short":"boolean"}},{"javadoc":"","static":false,"name":"uncaughtException","comment":"","params":[{"type_long":"java.lang.Thread","name":"thread","comment":"","type_short":"Thread"},{"type_long":"java.lang.Throwable","name":"t","comment":"","type_short":"Throwable"}],"throws":[],"return":{"type_long":"void","comment":"","type_short":"void"}}],"name":"ConcurrentTestExceptionHandler","comment":"A class which handles exceptions that occur off of the main test thread.  Exceptions can be\n reported to this class, which will later be checked by AbstractGenericTest.","fields":[]}
